students/Ben_freeman/lesson09/Mailroom_OO/mailroom_OO.py

```diff
@@ -25,17 +24,21 @@ def donor_donations(self):
         """counts the number of all of the donations of a particular donor"""
         return len(self.donor)
 
-    def __str__(self):
+    def message(self):
         """our default thank you message"""
-        return f"\nThank you {self.name} for your donation of ${self.donor_sum:.2f}, your donations make the work "\
-            f"of the 'American society for taking donations' possible.\n\n""Sincearly,\n\n""A low paid intern\n "
+        return f"\nThank you {self.name} for your donations totaling ${self.donor_sum:.2f}, your donations make the " \
+            f" work of the 'American society for taking donations' possible.\n\n""Sincearly,\n\n""A low paid intern\n "
 
+    def update_donor(self, new_donate):
+        """adds a donation to an existing donors profile"""
+        self.donor.append(new_donate)
```

Comment:
naming is crucial 🙂 

I didn't notice it last time, but this time reading the code I was lost at what `donor` attribute is.

The attribute name actually should be `donations`. I should be able to understand attributes, functions, classes without having to fully read the code. Just the name alone should tell me a lot about what the attribute is or what the function is expected to do.

Reply:
Will do.